.babelrc

```diff
@@ -3,78 +3,78 @@
     "development": {
```

Comment:
We could move this file to a `.js` file to remove all the duplication we have between the different environment.

Reply:
I actually find it more clear to have the exhaustive configuration hard coded in a file, as opposed to overriding it programmatically for specific environments.

Is configuration duplication a problem for you?

Note that if we go for a `babel.config.js` file, we'd better off do it in another PR because there are a lot of stacked PRs relying on that one.

Reply:
Yep I find it harder to read rather than a plain JS file. It's easy to forgot a place to add / update / remove a plugin or a preset, etc... We can do this change later or not.